python/arcticdb/storage_fixtures/azure.py

```diff
@@ -128,23 +142,38 @@ class AzuriteStorageFixtureFactory(StorageFixtureFactory):
     account_name = "devstoreaccount1"
     account_key = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
 
-    # Default cert path is used; May run into problem on Linux's non RHEL distribution
-    # See more on https://github.com/man-group/ArcticDB/issues/514
-    ca_cert_path = ""
-
     enforcing_permissions = False
     """Set to True to create AzureContainer with SAS authentication"""
 
-    def __init__(self, port=0, working_dir: Optional[str] = None):
+    def __init__(self, port=0, working_dir: Optional[str] = None, ssl: Optional[bool] = True):
         self.port = port or get_ephemeral_port(0)
-        self.endpoint_root = f"http://{self.host}:{self.port}"
+        self.http_protocol = "http" if platform.system() == "Windows" or "TEST_IN_ANALYSIS_WORKFLOW" in os.environ or ARCTICDB_USING_CONDA or not ssl else "https"
+        self.endpoint_root = f"{self.http_protocol}://{self.host}:{self.port}"
         self.working_dir = str(working_dir) if working_dir else mkdtemp(suffix="AzuriteStorageFixtureFactory")
+        
+        # Enable https on azurite needs to set ca cert; CA_cert_path cannot be set on Windows platform
```

Comment:
this (excellent) comment would make more sense up where `self.http_protocol` is set